playground/frontend/playground_components/lib/src/services/analytics/google_analytics_service.dart

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:usage/usage_html.dart';
+
+import 'event.dart';
+import 'generic/categories.dart';
+import 'generic/events.dart';
+import 'service.dart';
+
+class GoogleAnalyticsService implements AnalyticsService {
+  late AnalyticsHtml _appAnalytics;
+
+  GoogleAnalyticsService({
+    required AnalyticsHtml appAnalytics,
+  }) {
+    _appAnalytics = appAnalytics;
+  }
+
+  @override
+  AnalyticsEvent? lastSentEvent;
+
```

Comment:
There should be more events common for the two projects. Cancelling comes to mind.

I suggest also adding feedback here. This way we make sure that both ToB and Playground handle the feedback the same way.